frontend/src/index.tsx

```diff
@@ -5,6 +5,21 @@ import App from './App';
 import * as serviceWorker from './serviceWorker';
 import { BrowserRouter as Router } from 'react-router-dom';
 import WebFont from 'webfontloader';
+// JSS
+import JssProvider from 'react-jss/lib/JssProvider';
+import { create, JSS } from 'jss';
```

Comment:
Have you tried the `classes` property with material-ui? I saw that was a way of overriding their styled (as in injecting classes that take priority) but I didn't test it yet.
Maybe that is an easier way?